wiki.moinmoin

```diff
@@ -318,6 +322,7 @@ TODO: - does have a test suite that runs at build time
 TODO:   - test suite fails will fail the build upon error.
 TODO: - does have a non-trivial test suite that runs as autopkgtest
 TODO: - if special HW does prevent build/autopkgtest is there a test plan, code, log provided?
+TODO: - if a test on this level does not make sense, is the overall solution (app+libs) extensively covered?
```

Comment:
I think we should define more on the TODO "on this level". I don’t have a good proposal though off hand.

Reply:
I have clarified that this is meaning "non-trivial test on this level" and that that is usually represented by "(the lib alone is only doing rather simple things)".
Furthermore I also clarified the other side of it, which is what could be an extensive
test via "i.e. via end to end autopkgtest"

Thanks @didrocks